EmpireLegends/StartingPhase.cpp

```diff
@@ -284,5 +309,24 @@ void StartingPhase::setupNonPlayers()
 void StartingPhase::startBidding()
 {
 	// Players place bids
-	BiddingFacility::placeBids(players);
+	string maxBidder = BiddingFacility::placeBids(players);
+	maxBidderFirst(maxBidder);
+}
+
+void StartingPhase::maxBidderFirst(string maxBidder)
+{
+	for (int i = 0; i < players.size(); i++)
+	{
+		auto temp = players[i];
+		if (temp->getName() == maxBidder)
+		{
+			players.erase(players.begin() + i);
+			players.insert(players.begin(), temp);
+			return;
+		}
+		else
```

Comment:
don't think you actually need this else. if the if statement doesn't catch it, it will automatically just move onto the next loop iteration